awsconfigs/apps/pipeline/s3-static/kustomization.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+bases:
+- ../base
+configMapGenerator:
+- name: pipeline-install-config
+  env: params.env
+  behavior: merge
+- name: workflow-controller-configmap
+  behavior: replace
+  files:
+  - config
+- name: ml-pipeline-ui-configmap
+  behavior: replace
+  files:
+  - viewer-pod-template.json
+- name: kubeflow-pipelines-profile-controller-code
+  behavior: replace
+  files:
+  - sync.py
+generatorOptions:
+  disableNameSuffixHash: true
+patchesStrategicMerge:
+- disable-minio-server-resources.yaml
```

Comment:
consolidate this file so it only shows up once?

Reply:
done